quickfixj-core/src/main/java/quickfix/mina/ssl/SSLFilter.java

```diff
@@ -49,4 +58,20 @@ public void setEnabledCipherSuites(String[] cipherSuites) {
     public void setCipherSuites(String[] cipherSuites) {
         super.setEnabledCipherSuites(cipherSuites);
     }
+
+    @Override
+    public void onPreAdd(IoFilterChain parent, String name, NextFilter nextFilter)
+        throws SSLException {
+
+        IoSession session = parent.getSession();
+        SocketAddress remoteAddress = session.getRemoteAddress();
+
+        if(remoteAddress instanceof InetSocketAddress) {
+            // activate the SNI support in the JSSE SSLEngine
+            log.info("activating TLS SNI support");

```

Comment:
I would log the remote peer address being set.

Reply:
debug or info ?

Reply:
I would keep the current info and just added the param.